### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by SilviaBarcellini**
add footer component, which appears on the bottom section of every page of the app.
it contains the following components:
- home icon (with link)
- favourites icon (with link)

### CODE REVIEW

**Comment by mewiemers** [comment]
Wow.. soo much commits in one pull request, maybe you can divide it into more pull requests?

File: components/footer/favs/favsBtn.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styles from "./favsBtn.module.css";
+
+export type FavsIconProps = {
+  imgSrc: string;
+};
+
+function FavsIcon({ imgSrc, ...props }: FavsIconProps) {
```
Comment (mewiemers) [inline components/footer/favs/favsBtn.tsx:7]: here is a warning, props are never used

File: components/footer/favs/favsBtn.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react/types-6-0";
+import FavsIcon, { FavsIconProps } from "./favsBtn";
+
+export default {
+  title: "Common/favsIcon",
```
Comment (mewiemers) [inline components/footer/favs/favsBtn.stories.tsx:6]: maybe this?
 ```suggestion
  title: "Common/FavsIcon",
```

File: styles/globals.css [hunk 3]
```diff
@@ -1,16 +1,32 @@
...
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, "Poppins",
+    sans-serif, Quicksand, Paytone One;
 }
 
 a {
   color: inherit;
   text-decoration: none;
 }
 
+:root {
```
Comment (mewiemers) [inline styles/globals.css:18]: is root working on this place after html etc.?

**Review by mewiemers — COMMENTED** [review]
Looks good, just a few typos and advices 🍰

File: styles/globals.css (hunk 3 above)
Comment (lmachens) [reply styles/globals.css:18]: Yes :)

**Comment by SilviaBarcellini** [comment]
> Wow.. soo much commits in one pull request, maybe you can divide it into more pull requests?

There was a concern with the pull requests. Learnt from my mistakes 🤯

File: components/footer/favs/favsBtn.tsx (hunk 1 above)
Comment (SilviaBarcellini) [reply components/footer/favs/favsBtn.tsx:7]: Removed, thanks!

File: styles/globals.css (hunk 3 above)
Comment (SilviaBarcellini) [reply styles/globals.css:18]: It was there as default setting.

**Pull request merged**